lib/core/src/lib/language-menu/language-picker.component.ts

```diff
@@ -37,7 +37,25 @@ import { MatIconModule } from '@angular/material/icon';
         </mat-menu>
     `
 })
-export class LanguagePickerComponent {
+export class LanguagePickerComponent implements AfterViewInit {
     @Output()
     public changedLanguage = new EventEmitter<LanguageItem>();
+
+    @ViewChild('langMenu')
+    menu: MatMenu;
+
+    @ViewChild(MatMenuItem)
+    menuItem: MatMenuItem;
+
+    @ViewChild(LanguageMenuComponent)
+    languageMenuComponent: LanguageMenuComponent;
+
+    ngAfterViewInit() {
+        const menuItems: MatMenuItem[] = this.languageMenuComponent.menuItems.filter((menuItem: MatMenuItem) => menuItem !== undefined);
```

Comment:
no need to set type for menuItem inside filter function because it's set automatically based on type of menuItems